src/core/Translator.cpp

```diff
@@ -117,40 +104,32 @@ bool Translator::installQtTranslator(const QStringList& languages, const QString
  */
 QList<QPair<QString, QString>> Translator::availableLanguages()
 {
-    const QStringList paths = {
-#ifdef QT_DEBUG
-        QString("%1/share/translations").arg(KEEPASSX_BINARY_DIR),
-#endif
-        resources()->dataPath("translations")};
-
     QList<QPair<QString, QString>> languages;
     languages.append(QPair<QString, QString>("system", "System default"));
 
     QRegularExpression regExp("^keepassx_([a-zA-Z_]+)\\.qm$", QRegularExpression::CaseInsensitiveOption);
-    for (const QString& path : paths) {
-        const QStringList fileList = QDir(path).entryList();
-        for (const QString& filename : fileList) {
-            QRegularExpressionMatch match = regExp.match(filename);
-            if (match.hasMatch()) {
-                QString langcode = match.captured(1);
-                if (langcode == "en") {
-                    continue;
-                }
-
-                QLocale locale(langcode);
-                QString languageStr = QLocale::languageToString(locale.language());
-                if (langcode == "la") {
-                    // langcode "la" (Latin) is translated into "C" by QLocale::languageToString()
-                    languageStr = "Latin";
-                }
-                QString countryStr;
-                if (langcode.contains("_")) {
-                    countryStr = QString(" (%1)").arg(QLocale::countryToString(locale.country()));
-                }
+    const QStringList fileList = QDir(resources()->dataPath("translations")).entryList();
+    for (const QString& filename : fileList) {
+        QRegularExpressionMatch match = regExp.match(filename);
+        if (match.hasMatch()) {
+            QString langcode = match.captured(1);
+            if (langcode == "en") {
+                continue;
+            }
 
-                QPair<QString, QString> language(langcode, languageStr + countryStr);
-                languages.append(language);
+            QLocale locale(langcode);
+            QString languageStr = QLocale::languageToString(locale.language());
+            if (langcode == "la") {
+                // langcode "la" (Latin) is translated into "C" by QLocale::languageToString()
+                languageStr = "Latin";
             }
+            QString countryStr;
+            if (langcode.contains("_")) {
+                countryStr = QString(" (%1)").arg(QLocale::countryToString(locale.country()));
+            }
+
+            QPair<QString, QString> language(langcode, languageStr + countryStr);
```

Comment:
Why define a new variable that isn't used anywhere else? Just append it to `languageStr`.